examples/cmd/benchmark.go

```diff
@@ -106,7 +106,11 @@ func runBenchmark(cmd *cobra.Command, _ []string) error {
 			return err
 		}
 		nanoTDFConfig.EnableECDSAPolicyBinding()
-		err = nanoTDFConfig.SetKasURL(fmt.Sprintf("http://%s/kas", "localhost:8080"))
+		if insecurePlaintextConn || strings.Contains(platformEndpoint, "http://") {
+			err = nanoTDFConfig.SetKasURL(fmt.Sprintf("http://%s/kas", "localhost:8080"))
```

Comment:
Wait, shouldn't this be based off `platformEndpoint` somehow?

Reply:
ya im not sure why were hardcoding it